sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -381,73 +396,72 @@ def get_test_params(cls, parameter_set="default"):
 
         Returns
         -------
-        params : dict or list of dict, default = {}
-            Parameters to create testing instances of the class
-            Each dict are parameters to construct an "interesting" test instance, i.e.,
-            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
-            `create_test_instance` uses the first (or only) dictionary in `params`
+        params : list of dict
+            Parameters to create testing instances of the class.
+            Each dict contains parameters to construct an "interesting" test instance,
+            i.e., `MyClass(**params)` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`.
         """
+        base_training_args = {
+            "num_train_epochs": 1,
+            "output_dir": "test_output",
+            "per_device_train_batch_size": 32,
+        }
+
+        base_config = {
+            "lags_sequence": [1, 2, 3],
+            "context_length": 2,
+            "prediction_length": 4,
+        }
+
         test_params = [
+            # General transformer-based test cases
             {
                 "model_path": "huggingface/informer-tourism-monthly",
                 "fit_strategy": "minimal",
-                "training_args": {
-                    "num_train_epochs": 1,
-                    "output_dir": "test_output",
-                    "per_device_train_batch_size": 32,
-                },
-                "config": {
-                    "lags_sequence": [1, 2, 3],
-                    "context_length": 2,
-                    "prediction_length": 4,
-                },
-                "deterministic": True,
-            },
-            {
-                "model_path": "huggingface/autoformer-tourism-monthly",
-                "fit_strategy": "minimal",
-                "training_args": {
-                    "num_train_epochs": 1,
-                    "output_dir": "test_output",
-                    "per_device_train_batch_size": 32,
-                },
-                "config": {
-                    "lags_sequence": [1, 2, 3],
-                    "context_length": 2,
-                    "prediction_length": 4,
-                    "label_length": 2,
-                },
+                "training_args": base_training_args,
+                "config": base_config,
                 "deterministic": True,
-            },
+            }
         ]
 
-        if _check_soft_dependencies("peft", severity="none"):
-            from peft import LoraConfig
+        if _check_soft_dependencies(
+            "transformers", severity="none"
+        ) and _check_soft_dependencies("torch", severity="none"):
```

Comment:
I don't think we really need to check `torch` separately, its already a requirement for `transformers`
```suggestion
        if _check_soft_dependencies("transformers", severity="none"):
```